src/assets/js/react/components/Help/HelpContainer.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import request from 'superagent'
+import { updateResult, deleteResult } from '../../actions/help'
+
+
+export const doHandleChange = data => ({
+  searchInput: data
+});
+
+export const toggleLoadingTrue = () => ({
+  loading: true
+});
+
+export const toggleLoadingFalse = () => ({
+  loading: false
+})
+
+export const fetchData = searchInput => {
+  // Request API call
+  return request.get(`https://gravitypdf.com/wp-json/wp/v2/v5_docs/?search=${searchInput}`);
+}
+
+
+class HelpContainer extends Component {
+
+  // Initialize component state
+  state = {
+    searchInput: '',
+    loading: false
+  }
+
+  onHandleChange = e => {
+    // Set loading to true
+    this.setState(doHandleChange(e.target.value));
+    // Set searchInput state value
+    this.searchInputLength(e.target.value);
+  }
+
+  searchInputLength = data => {
+    if (data.length > 3) {
+      // Set loading to true
+      this.setState(toggleLoadingTrue);
+      // Request API call
+      fetchData(data)
+        // If request is successful
+      .then(res => {
+        // Pass data into redux action
+        this.props.updateResult(res.body)
+        this.setState(toggleLoadingFalse)
+      })
+      // Catch if something went wrong in the call
+      .catch(err => {
+        console.log('action err -', err)
+      });
+    } else {
+      this.props.deleteResult();
+    }
+  }
+
+  displayResult = () => {
+    const { loading, searchInput } = this.state;
+    const { helpResult } = this.props;
+    let searchResult;
+    let items;
+
+    // Check if search result is not emplty or loading is true
+    if (helpResult.length > 0 || loading) {
+      // map the search result
+      items = (
+        helpResult.map((item, index) => (
+          <li key={index}>
+            <a href={item.link} >{item.title.rendered}</a>
+            <div className="except">
+              <div>
+                {/* Parse html */}
+                <div dangerouslySetInnerHTML={{__html: item.excerpt.rendered}} />
+              </div>
+            </div>
+          </li>
+        ))
+      )
+    } else {
+      items = (
+        <li>It doesn't look like there are any topics related to your issue.</li>
+      )
+    }
+   
+    // Check if state searchInput length is greatar than 3 then display items
+    if (searchInput.length > 3) {
+      searchResult = (
+        <React.Fragment>
+          <h3 className="hndle">
+            <span>Gravity PDF Documentation</span>
+            { loading ? <span className="spinner is-active"></span> : null  }
```

Comment:
Use the `<Spinner />` component for this.

Reply:
Fixed